vello_hybrid/vello_api/LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright 2020 the Vello Authors
```

Comment:
If we expect not to copy too much non-trivial code from current-vello, it _might_ be possible to go with the "no attribution" version. E.g., `color` has: https://github.com/linebender/color/blob/ba0bbaea84baf9e578429aacd3c39aae6ca7e92a/LICENSE-MIT (@DJMcNab).

Reply:
I think the attribution is fine to have. Something which my previous claims had missed, was that to merge multiple MIT licenses, it's *probably* valid to just have one block with multiple attributions, and the single block of license text.
That is, if we're taking code from a project with a different attribution here (e.g. from swash for #823), we can add their attributions to the license.

For reference, Rust has moved back to the form with attribution.

Reply:
> to merge multiple MIT licenses, it's probably valid to just have one block with multiple attributions, and the single block of license text

That's certainly how I'd read "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software." If that's indeed the case, it's much less painful to merge.

Reply:
Yes, I think we're all agreed that would be OK. There has been some confusion and churn around this, see e.g. the Rust project moving to the "no-attribution" MIT license: https://github.com/rust-lang/rust/commit/2a8807e889a43c6b89eb6f2736907afa87ae592f.

(And last December reintroducing the attribution string, but without a year: https://github.com/rust-lang/rust/commit/f9c16997dc016a3ef1456f56df2ab564a1c48cb2.)